providers/digitalocean_test.go

```diff
@@ -98,8 +100,9 @@ func TestDigitalOceanProviderGetEmailAddress(t *testing.T) {
 	bURL, _ := url.Parse(b.URL)
 	p := testDigitalOceanProvider(bURL.Host)
 
+	ctx := context.TODO()
```

Comment:
Please do the others in this file too
```suggestion
	ctx := context.Background()
```